doc/src/explore/sui-explorer.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Using the Sui Explorer
+---
+
+# Using Sui Explorer
+The [Sui Explorer](https://explorer.devnet.sui.io/) lets you view data about transactions and activity on the Sui network. In addition to viewing activity on the network, you can use the Explorer to:
+ * View up-to-date information about the activity and metrics on the Sui network.
+ * Look up, verify, and track your assets and contracts.
+ * Utilize fast, reliable, and transparent debugging and auditing data to help identify and resolve issues. 
+ * Get go-to-definition support for all smart contracts, referred to as packages in Sui.
+ * View validators and geographic locations of full nodes that are currently active.
```

Comment:
We use "fullnodes" (all one word) here:
https://docs.sui.io/devnet/build/fullnode

Reply:
this was changed by Bauer in source doc